pkg/gds/admin/v2/auth.go

```diff
@@ -0,0 +1,28 @@
+package admin
+
+import (
+	"errors"
+	"strings"
+
+	"github.com/gin-gonic/gin"
+)
+
+const (
+	bearer        = "Bearer "
```

Comment:
Just checking that the extra whitespace char was intentional

Reply:
It is intentional in this case, but good catch!